src/neptune/internal/backends/neptune_backend.py

```diff
@@ -211,6 +211,12 @@ def get_float_series_attribute(
     ) -> FloatSeriesAttribute:
         pass
 
+    @abc.abstractmethod
+    def get_float_series_values_proto(
```

Comment:
I think this is not needed (no point of extending the abstract class/interface) as this is a part of implementation detail what protocol/endpoint to use. Could we just have two private methods as a part of `HostedNeptuneBackend` and keep the interface the same as it was before?

Reply:
Yes we can do that. The only issue with that it requires client config support flag which was out of scope of this PR.

For now I have implemented proto support bloating `NeptuneBackend` method pool in next PR (attached on top of this one) https://github.com/neptune-ai/neptune-client/pull/1619

```
HostedNeptuneBackend now for both fetching string and float series values has methods:
get_*_series_values_json: that works as before with json dto
get_*_series_values_proto: with new implementation using protobuf
get_*_series_values: chooses proto or json implementation based on client config
```

Reply:
> This change requires client config proto support flag which was out of scope of this PR

You can just add a private method in `HostedNeptuneBackend` without any calls of it for now. No need to worry about client config in this PR as you already mentioned.

Reply:
At the end I just implmented `*_proto` methods as private methods of `HostedNeptuneBacked` without changing `NeptuneBackend` interface